authenticationsdk/core/MerchantConfiguration.py

```diff
@@ -256,6 +266,91 @@ def set_p12KeyFilePath(self, value):
+        if not (value.get('enableResponseMleGlobally') is None):
+            self.enableResponseMleGlobally = value['enableResponseMleGlobally']
+        else:
+            self.enableResponseMleGlobally = False
+
+    def set_responseMleKID(self, value):
+        if not (value.get('responseMleKID') is None):
+            self.responseMleKID = value['responseMleKID']
+
+    def set_responseMlePrivateKeyFilePath(self, value):
+        if not (value.get('responseMlePrivateKeyFilePath') is None):
+            self.responseMlePrivateKeyFilePath = value['responseMlePrivateKeyFilePath']
+
+    def set_responseMlePrivateKeyFilePassword(self, value):
+        if not (value.get('responseMlePrivateKeyFilePassword') is None):
+            self.responseMlePrivateKeyFilePassword = value['responseMlePrivateKeyFilePassword']
+
+    def set_responseMlePrivateKey(self, value):
+        if not (value.get('responseMlePrivateKey') is None):
+            self.responseMlePrivateKey = value['responseMlePrivateKey']
+
+    def set_requestMleKeyAlias(self, value):
+        if not (value.get('requestMleKeyAlias') is None) and value.get('requestMleKeyAlias').strip():
+            self.requestMleKeyAlias = value['requestMleKeyAlias'].strip()
+        elif not (value.get('mleKeyAlias') is None) and value.get('mleKeyAlias').strip():
+            self.requestMleKeyAlias = value['mleKeyAlias'].strip()
+        else:
+            self.requestMleKeyAlias = GlobalLabelParameters.DEFAULT_MLE_ALIAS_FOR_CERT
+            
+    def get_requestMleKeyAlias(self):
+        return self.requestMleKeyAlias
+
+    def set_mleForRequestPublicCertPath(self, value):
+        if not (value.get('mleForRequestPublicCertPath') is None):
+            self.mleForRequestPublicCertPath = value['mleForRequestPublicCertPath']
+
+    def set_internalMapToControlMLEonAPI(self):
+        if self.mapToControlMLEonAPI is not None:
+            # Initialize internal maps
+            self.internalMapToControlRequestMLEonAPI = {}
+            self.internalMapToControlResponseMLEonAPI = {}
+
+            for api_name, value in self.mapToControlMLEonAPI.items():
+                if isinstance(value, bool):
+                    # If value is boolean, it only applies to request MLE
+                    self.internalMapToControlRequestMLEonAPI[api_name] = value
+                elif isinstance(value, str) and '::' in value:
+                    # Format: "requestMLE::responseMLE"
+                    parts = value.split('::', 1)  # Split at first occurrence of '::'
+                    request_mle = parts[0].strip()
+                    response_mle = parts[1].strip() if len(parts) > 1 else ""
+
+                    # Set request MLE value if provided
+                    if request_mle:
+                        self.internalMapToControlRequestMLEonAPI[api_name] = request_mle.lower() == 'true'
+
+                    # Set response MLE value if provided
+                    if response_mle:
```

Comment:
verify how it will behave when response_mle = "". 

for case value="true::" --> expectation is it should not set the value for responseMle